init_docs/Text.json

```diff
@@ -842,7 +842,8 @@
   "viplink_delete":"Are you sure that you want to delete this link?",
   "viplinks":"Vip Links",
   "Create":"Create Link",
-  "Link_Coppied":"Link copied to your clipboard",
+  "Link_Copied":"Link copied to your clipboard",
```

Comment:
`Link_Coppied` needs to be changed also in `app/nation/app/views/VipLinksView.js`